app/javascript/components/pxe-servers-form/pxe-server-form.schema.js

```diff
@@ -76,6 +77,83 @@ const createSchema = isEditing => ({
           value: 'smb://',
           label: 'Samba',
         }],
+      }, {
+        component: componentTypes.TEXT_FIELD,
+        name: "uri-field",
+        condition: {
+          when: 'uri',
+          isNotEmpty: true,
+        },
+        inputAddon: {
+          before: {
+            fields: [
```

Comment:
Instead of these buttons, the select should be here directly.

Reply:
This review still stands, why aren't you putting the select into the input group instead? @ZitaNemeckova

Reply:
@skateman Because I'm not replacing `input-with-dynamic-prefix` and using `inputAdon`.

Reply:
And why is that? We could get rid of a custom component.

Reply:
Okay, I kinda see the point. Just trying to minimize the number of custom components because of DDFv2.

Reply:
Because this is a quick bug fix that unblockes you with minimal changes to the form. Replacing `input-with-dynamic-prefix` with something default is not trivial thing in DDF. I tried and failed. You can give it a try if you want :)